src/components/SearchBar.tsx

```diff
@@ -12,80 +12,73 @@ import SearchBarNativeComponent, {
 } from '../fabric/SearchBarNativeComponent';
 
 export const NativeSearchBar: React.ComponentType<SearchBarProps> &
-  typeof NativeSearchBarCommands = SearchBarNativeComponent as any;
+  typeof NativeSearchBarCommands =
+  SearchBarNativeComponent as unknown as React.ComponentType<SearchBarProps> &
+    SearchBarCommandsType;
 export const NativeSearchBarCommands: SearchBarCommandsType =
-  SearchBarNativeCommands as any;
+  SearchBarNativeCommands as SearchBarCommandsType;
+
+type ViewRef = React.ElementRef<typeof NativeSearchBar>;
 
 type SearchBarCommandsType = {
-  blur: (viewRef: React.ElementRef<typeof NativeSearchBar>) => void;
-  focus: (viewRef: React.ElementRef<typeof NativeSearchBar>) => void;
-  clearText: (viewRef: React.ElementRef<typeof NativeSearchBar>) => void;
-  toggleCancelButton: (
-    viewRef: React.ElementRef<typeof NativeSearchBar>,
-    flag: boolean
-  ) => void;
-  setText: (
-    viewRef: React.ElementRef<typeof NativeSearchBar>,
-    text: string
-  ) => void;
-  cancelSearch: (viewRef: React.ElementRef<typeof NativeSearchBar>) => void;
+  blur: (viewRef: ViewRef) => void;
+  focus: (viewRef: ViewRef) => void;
+  clearText: (viewRef: ViewRef) => void;
+  toggleCancelButton: (viewRef: ViewRef, flag: boolean) => void;
+  setText: (viewRef: ViewRef, text: string) => void;
+  cancelSearch: (viewRef: ViewRef) => void;
 };
 
-class SearchBar extends React.Component<SearchBarProps> {
-  nativeSearchBarRef: React.RefObject<SearchBarCommands>;
-
-  constructor(props: SearchBarProps) {
-    super(props);
-    this.nativeSearchBarRef = React.createRef();
-  }
+function ImplSearchBar(props: SearchBarProps, ref: Ref<SearchBarCommands>) {
```

Comment:
In my nitpicks I will insist here on using suffix instead of prefix.
 
```suggestion
function SearchBarImpl(props: SearchBarProps, ref: Ref<SearchBarCommands>) {
```

Reply:
I don't think `ImplSearchBar` would be good name. How about just `SearchBar`?

Reply:
@alduzy I saw your commit about the name change 😄 but what do you think about it?

Reply:
Good idea, especially since the export doesn't need name, previously the outer function declaration was SearchBar

Reply:
https://github.com/software-mansion/react-native-screens/pull/2132/commits/33cc656967c6ab70262008ec3f79ac2c9ad1a550